models/UserManager.php

```diff
@@ -0,0 +1,315 @@
+<?php
+
+class UserManager extends Model
+{
+    // Fonction permettant de vérifier que la session est bien active sous le role administrateur
+    function admin(){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [admin accesses the super-global variable $_SESSION.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)